bankroll/model/modelconverter.py

```diff
@@ -0,0 +1,16 @@
+import pandas as pd
```

Comment:
This should be an optional dependency, so we don't crash anyone who doesn't have it installed.

Reply:
✅ pandas are now optional 🐼